pulse_actions/handlers/talos.py

```diff
@@ -28,7 +29,10 @@ def on_event(data, message, dry_run):
     payload = data["payload"]
     status = payload["status"]
     buildername = payload["buildername"]
-    info = get_buildername_metadata(buildername)
+    try:
+        info = get_buildername_metadata(buildername)
+    except MissingBuilderError:
+        return
```

Comment:
If we use a return we don't get to line 72:
https://github.com/mozilla/pulse_actions/pull/84/files#diff-942de6cda6cf1ef14393a1ba78e96082R72

That would leave the message inside of the pulse queue and never be cleared. Thus, every time the Heroku dynos would reboot or a new deployment happened we would process the message again.

Could you also move the try block after the line below?